examples/plot_unicode.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""
+=======================================
+Únîcödè support in documentation string
+=======================================
+
+In this example we include Unicode characters that are so required this days.✓
+"""
+
+# Code source: Óscar Nájera
+# License: BSD 3 clause
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+N = 50
+x = np.linspace(-1,2,50)
```

Comment:
> I am one who thinks that Unicode in source code is a bad thing.

I think this more a matter of timing. As I said with python 3 having unicode as default implies the need for foreign languages support and that it is quite an issue. And this bug fix is not even necessary in python 3.
The reason I'm proposing this is not because of having unicode in the code. That already worked as you can find by checking that my examples included my Spanish accented name. And I agree that the source code should be the most plain text possible. I propose this because it is the main docstring of the example that could not include foreign characters. Docstrings are aimed for a broader audience than the programmers, so that they are most certainly are in the target audience most conforming setup, there every body would be able to read.